db/re/item_db_etc.yml

```
@@ -9540,10 +9540,11 @@ Body:
       DropEffect: CLIENT
     Script: |
       .@rate = ((getrefine()>14)?7:5);
+      .@duration = ((getrefine()>14)?10000:15000);
```

The timer is not a duration but an interval
```suggestion
      .@duration = ((getrefine()>14)?10000:5000);
```

---

Thanks